functions.php

```diff
@@ -474,3 +474,15 @@ function nightingale_learndash_admin_fix() {
 if ( ! is_admin() ) {
 	require get_template_directory() . '/inc/dynamic-blocks.php';
 }
+
+/**
+ * For security prevent RSS feed disclosing author usernames
+ */
+function nhsla_check_author( $display_name ) {
```

Comment:
can this be renamed to nightingale_check_author pls for consistency?

Reply:
Done